cabby/geo/visualization/database.py

```diff
@@ -0,0 +1,49 @@
+
+from datetime import datetime
+from flask import *
```

Comment:
Don't import all from a module. Instead 'import flask' and then use flax.foo, etc., or use 'from flax import foo' for each thing you need.